cms/models.py

```diff
@@ -1059,6 +1060,7 @@ def get_context(self, request, *args, **kwargs):
                 request.build_absolute_uri("///"),
                 static("images/certificates/share-image.png"),
             ),
+            "share_text": "I just earned a certificate",
```

Comment:
```suggestion
            "share_text": "I just earned a certificate in {}".format(
                self.bootcamp_run_name
            ),
```

I think we should include `Bootcamp Run Name`, Contextually, similar to what we are doing in other application https://github.com/mitodl/mitxpro/blob/b4eed13024fc9438a25b05d47a2292a20e55dbf7/cms/models.py#L1597.

Reply:
updated.